Makefile

```diff
@@ -1,3 +1,4 @@
+SHELL := /bin/bash
```

Comment:
More portable: `SHELL := /usr/bin/env bash`. Can you change this please? (The other Makefiles do this).